[2025-10-26 16:01:03] ERROR: ‚ùå Database synchronization failed: connect ETIMEDOUT {"name":"SequelizeConnectionError","parent":{"errorno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","fatal":true},"original":{"errorno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","fatal":true},"stack":"SequelizeConnectionError: connect ETIMEDOUT\n    at ConnectionManager.connect (/Users/jddnk/Desktop/Holmesglen (2025)/Semester 2/BIS311/ASS1/photostore-api1/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:102:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (/Users/jddnk/Desktop/Holmesglen (2025)/Semester 2/BIS311/ASS1/photostore-api1/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)\n    at async /Users/jddnk/Desktop/Holmesglen (2025)/Semester 2/BIS311/ASS1/photostore-api1/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:174:32\n    at async ConnectionManager.getConnection (/Users/jddnk/Desktop/Holmesglen (2025)/Semester 2/BIS311/ASS1/photostore-api1/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:197:7)\n    at async /Users/jddnk/Desktop/Holmesglen (2025)/Semester 2/BIS311/ASS1/photostore-api1/node_modules/sequelize/lib/sequelize.js:305:26\n    at async MySQLQueryInterface.tableExists (/Users/jddnk/Desktop/Holmesglen (2025)/Semester 2/BIS311/ASS1/photostore-api1/node_modules/sequelize/lib/dialects/abstract/query-interface.js:102:17)\n    at async Product.sync (/Users/jddnk/Desktop/Holmesglen (2025)/Semester 2/BIS311/ASS1/photostore-api1/node_modules/sequelize/lib/model.js:939:21)\n    at async Sequelize.sync (/Users/jddnk/Desktop/Holmesglen (2025)/Semester 2/BIS311/ASS1/photostore-api1/node_modules/sequelize/lib/sequelize.js:377:9)"}
